views/incubator/omniledger.tpl

```diff
@@ -0,0 +1,118 @@
+            <h3>Consensus</h3>
+            <img src="/resources/incubator/images/omniledger/distributed-ledger.png" width="33%"
+                 style="float: right; padding: 1em;"/>
+            There are other blockchains than Bitcoin, but they all solve the problem of
+            <strong>consensus</strong> for decentralized systems. Reaching a consensus between
+            different entities is not easy, as some of the entities can try to cheat.
+            So the conception of blockchains always takes into account that some of the
+            participants will try to cheat. If too many are cheating, the consensus will fail,
+            or invalid transactions might get accepted, leading to a failure of the
+            decentralization.
+        </td>
+    </tr>
+</table>
+
+<h2>OmniLedger</h2>
+
+<p>
+    When building blockchains, there are a number of technical challenges to overcome.
+    OmniLeger solves the following challenges:
+</p>
+<ul>
+    <li><strong>Speed</strong> - if many people want to use the system, it needs to be fast. Speed is measured
+        in two ways: transactions per second, and confirmation delay
+    </li>
+    <ul>
+        <li><strong>Transactions per second</strong> - define how many money transfers can be processed by the system.
+            VISA can process up to 10'000 transactions in a second, while Bitcoin can do about 7. Depending on the
+            attacker, OmniLedger can go well beyond 10'000 transactions per second.
+        </li>
+        <li><strong>Confirmation delay</strong> - how long you have to wait to be sure your money transfer arrived.
+            VISA takes about 5 seconds, Bitcoin takes about 1h. OmniLedger transactions take between 2 seconds and 1
+            minute.
+        </li>
+    </ul>
+    <li><strong>Attack resistance</strong> - if you receive money in the system, you want to make sure it
+        stays there, and does not disappear all of a sudden. OmniLedger can proof that even with the high throughput it
+        has, it is still save and will not allow an attacker to spend the same money twice.
+    </li>
+</ul>
+<p>
+    In addition to these challenges, we implemented
+    <a href="https://factory.c4dt.org/showcase/labs/DEDIS/calypso">Calypso</a>,
+    which is a <strong>Secret storage</strong> service.
+    This is needed, because today's blockchains are often world-readable, so it is important
+    that you still can store secret data without everybody being able to read it.
+    Calypso is not part of the original OmniLedger paper, but uses the
+    OmniLedger blockchain implementation.
+    </li>
+</p>
+
+<h2>Links to demos</h2>
+
+<p>OmniLedger has already been used for a number of years, and there are a lot of projects that
+    interact with this blockchain.
+    Together with the <a href="https://dedis.ch">DEDIS</a> lab, C4DT is running a test-network for
+    more than a year now, and it is used on a daily basis for different services.</p>
```

Comment:
Suggestion: instead of "for more than a year now", use something like "since XXX", so as to avoid having to update it in time... :)

Reply:
March 2019...